onnxruntime/contrib_ops/cuda/bert/skip_layer_norm_impl.cu

```diff
@@ -28,9 +35,11 @@ namespace onnxruntime {
 namespace contrib {
 namespace cuda {
 
+constexpr float one = 1.0;
+
 template <typename T, unsigned TPB>
 __global__ void SkipLayerNormKernelSmall(
```

Comment:
Can we delete SkipLayerNormKernelSmall now, as it is the same as SkipLayerNormKernelSmallVec<T, TPB, 1>? Maybe just rename SkipLayerNormKernelSmallVec to SkipLayerNormKernelSmall?